lighthouse/tests/test_lighthouse.py

```diff
@@ -33,11 +33,27 @@ def test_check(aggregator, instance):
     tags.append("url:{0}".format(instance["url"]))
     tags.append("name:{0}".format(instance["name"]))
```

Comment:
It would be nice to have a test using `form_factor` as well as a test using the new `urls` config option.